Libraries/Components/TextInput/TextInput.js

```diff
@@ -1392,6 +1400,7 @@ const ExportedForwardRef: React.AbstractComponent<
       allowFontScaling={allowFontScaling}
       rejectResponderTermination={rejectResponderTermination}
       underlineColorAndroid={underlineColorAndroid}
+      editable={!readOnly}
```

Comment:
We're going to have `readOnly` take priority over `editable`. Thanks

Reply:
Sounds good, I've just updated the logic to match this.